cleanlab_studio/studio/studio.py

```diff
@@ -153,7 +154,7 @@ def apply_corrections(self, cleanset_id: str, dataset: Any, keep_excluded: bool
                 # instead, use original JSON, which uses null values where it's not specified
                 udf(lambda original, clean: original if check_none(clean) else clean)(
                     both[label_column],
-                    "clean_label",
+                    "corrected_label",
```

Comment:
Were you able to check if we can make a similar change as in the pandas case here to remove the hacky code here?